modules/enumeratum/src/main/scala/pureconfig/module/enumeratum.scala

```diff
@@ -0,0 +1,30 @@
+package pureconfig.module
+
+import _root_.enumeratum.values._
+import _root_.enumeratum._
+import pureconfig.ConfigConvert
+
+import scala.util.Try
+
+package object enumeratum {
+  implicit def enumeratumConfigConvert[A <: EnumEntry](implicit enum: Enum[A]): ConfigConvert[A] =
+    ConfigConvert.stringConvert[A](name => Try(enum.withName(name)), _.toString)
```

Comment:
Just a suggestion: I would consider importing `ConfigConvert.stringConvert` just to remove a little boilerplate from each of these lines.